zero_bin/rpc/Cargo.toml

```diff
@@ -14,9 +14,11 @@ __compat_primitive_types = { workspace = true }
 alloy.workspace = true
 anyhow = { workspace = true }
 clap = { workspace = true }
+env_logger = { workspace = true }
```

Comment:
I don't think this is needed?

Reply:
Didn't expect the use of different logging mechanisms between crates.  Nuked.